parsing/asttypes.mli

```diff
@@ -25,6 +25,7 @@ type rec_flag = Nonrecursive | Recursive
 
 type direction_flag = Upto | Downto
 
+(* Order matters, used in polymorphic comparison *)
```

Comment:
Urgh!  If you can easily find such sites, it would be worth documenting them here (and later, one should fix that...).